modules/lightmapper_rd/lightmapper_rd.h

```diff
@@ -262,16 +262,18 @@ class LightmapperRD : public Lightmapper {
 		float albedo_bandwidth;
 		float normal_bandwidth;
 
+		int half_search_window;
+
```

Comment:
When passing parameters to the GPU, we try to keep things aligned to packs of 16 bytes visually to avoid alignment issues between C++ and GLSL.

While the code here does work, I'd recommend getting rid of the empty line after it and pack it together with the next group.